### PR TITLE
Include a link to ChainerX product backlog

### PR DESCRIPTION
**Pull request by hvy**
Part of https://github.com/chainer/chainer/issues/6629.

The ChainerX product backlog is now made public.

### CODE REVIEW

File: chainerx.md [hunk 1]
```diff
@@ -4,14 +4,20 @@
 It is implemented almost purely in C++ where the C++ implementations are exposed via Python bindings, allowing Chainer to make use of it.
 It does in other words **not** replace Chainer. It aims to instead improve the performance of Chainer in terms of speed by reducing the Python overhead.
 
-This guide is aimed toward users familiar with the Chainer interface but want to improve their training/inference speed, using ChainerX.
-It explains how to install it, the motivation behind it and how to modify your existing Chainer code to use ChainerX.
+The first section is the product backlog of ChainerX listing its milestones.
+Following guides ares aimed toward users familiar with the Chainer interface but want to improve their training/inference speed, using ChainerX.
+They explain how to install it, the motivation behind it and how to modify your existing Chainer code to use ChainerX.
 
+- [Product Backlog](#product-backlog)
```
Comment (niboshi) [inline chainerx.md]: Product backlog is not for users. I think it should come last.

(I think this document needs an overall cleanup, but peprhaps it's another issue.)

Comment (hvy) [reply chainerx.md]: Ok, I'll move it. Discussed it with @beam2d offline that we probably can remove this file altogether and copy the necessary content to the official docs in a separate PR.

Comment (hvy) [reply chainerx.md]: Fixed.

**Comment by niboshi** [comment]
Jenkins, test this please

**Comment by niboshi** [comment]
LGTM

**Pull request merged**